tests/unit/qunit/test-loader-test.js

```diff
@@ -60,6 +60,30 @@ test('loads all test modules by default', function(assert) {
   ]);
 });
 
+test('loads all test modules when testem object is not available', function(assert) {
+  const testemObject = this.testem;
+  this.testem = undefined;
+  const testLoader = new EmberExamTestLoader(
+    this.testem,
```

Comment:
is there a reason we don't just pass `undefined` here?

Reply:
we could. My initial thought was that it would be more easy to read if we pass this.testem after assign value to undefined. But I'm opened to any opinions!